django/core/cache/backends/memcached.py

```diff
@@ -2,12 +2,23 @@
 
 import pickle
 import re
+import socket
 import time
 
+import six
```

Comment:
may I know what is the usage of six here?

Reply:
@auvipy six dependency removed.